scripts/base/protocols/smb/main.zeek

```diff
@@ -225,7 +225,12 @@ function write_file_log(state: State)
 			else
 				add state$recent_files[file_ident];
 			}
-		
+
+		local log_f = copy(f);
+		if (!log_f?$ts){
+			log_f$ts = network_time()
```

Comment:
The log_f variable is never used for anything here or in the other changes.  This set of changes wouldn't actually do anything.

Reply:
I realized now that I have missed changing the variable while writing into the log file. All three changes in the commit needed to be logged as shown below.

Log::write(FILES_LOG, log_f);

I am happy to do another PR for it and submit the pcap with it too.